src/nuis/measurement/ProjectionHelper.cxx

```diff
@@ -233,90 +228,70 @@ int Record::FillBin(int i, double w) {
 }
 
 
-int Record::FillBin(const std::vector<double>& indep_vals, double w) {
+int ProjectionHelper::FillBin(const std::vector<double>& indep_vals, double w) {
     int i = GetBin(indep_vals);
     return FillBin(i, w);
 }
 
-void Record::FillTally(const double w) {
-    total_mc_tally += w;
-}
-
-uint32_t Record::GetMCCounts(const uint32_t i) {
+uint32_t ProjectionHelper::GetMCCounts(const uint32_t i) {
```

Comment:
if we're going to use the fixed width types for the bin index, we should be consistent throughout the API.